### DISCUSSION TITLE
Usage in rspack/rsbuild projects?

### DISCUSSION THREAD

**Discussion by dangmai** [anolilab/unplugin-favicons#70] [Q&A] [answered]
Hello there 👋 
Thanks for creating this! I'm trying it out for a rsbuild project, and I can see an example for it in this repo: https://github.com/anolilab/unplugin-favicons/tree/main/examples/rspack However, when I look at the files distributed in NPM for this package, I don't see the `rspack` plugin being distributed: https://www.npmjs.com/package/@anolilab/unplugin-favicons?activeTab=code 
Am I missing something? Would appreciate any help.

**Comment by prisis** [comment]
Hey @dangmai, i sorry for the late response...

Looks like i forgot to export the rsbuild file in package.json

I will work today on this lib, to give it a bit more love :)

**Reply by dangmai** [reply]
Thank you, I'm looking forward to it!

**Comment by prisis** [comment, marked as answer]
I just released an alpha version with the fix 
https://github.com/anolilab/unplugin-favicons/releases/tag/v1.0.6-alpha.1
